proposals/openebs.adoc

```diff
@@ -0,0 +1,137 @@
+
+**Statement on alignment with CNCF mission:**
+
+The primary purpose of this submission is to enable broader collaboration thanks to the role of the CNCF as a neutral home for open source projects that add value to the overall cloud native ecosystem.  In particular as OpenEBS and OpenEBS components appear to be of interest to multiple otherwise competitive technology and service providers, the independence of governance of the CNCF seems likely to unlock additional innovation and collaboration.
+
+OpenEBS is aligned with our understanding of the CNCF mission.  
+
+* Extensible and open architecture based upon Kubernetes
+* Horizontally scalable with Kubernetes nodes
+* Integrated with other CNCF projects like Kubernetes, Prometheus, etcd, and others
+* Additionally, the OpenEBS community attempts to be extremely inclusive and is one of the more active CNCF related projects in India, actively working with technical universities to introduce interested students to Open Source practices and projects, including OpenEBS, Prometheus, WeaveWorks, Litmus (also sponsored by MayaData for chaos engineering and end to end testing) and other projects.  
+
+We hope to continue to broaden OpenSource understanding and adoption and believe that OpenEBS becoming a part of the CNCF will further these objectives.
+
+*Roadmap:* GitHub (https://github.com/openebs/openebs/milestones)
+
+*Sponsors from TOC:* Xiang Li, Alexis Richardson
+
+*Preferred maturity level:* Sandbox
+
+*License:* Apache License v2.0
+
+*Source control and External Dependencies:* 
+
+OpenEBS Source code is split across multiple repositories, some of which are listed below. The complete details of the repositories and their dependencies can be found in: https://github.com/openebs/openebs/blob/master/contribute/design/README.md
+
+* https://github.com/openebs/openebs
+* https://github.com/openebs/maya
+* https://github.com/openebs/jiva
+* https://github.com/openebs/cstor
+* https://github.com/openebs/istgt
+* https://github.com/openebs/node-disk-manager
+
+*Initial Committers and Founding Maintainers:* 
+
+ * Kiran Mova @kmova (MayaData)
+ * Amit Kumar Das @AmitKumarDas (MayaData)
+ * Karthik Satchitanand @ksatchit (MayaData)
+
+*Additional Maintainers:*
+
+ * Murat Karslioglu @muratkars (MayaData)
+ * Jeffry Molanus @gila (MayaData)
+ * Vishnu Itta @vishnuitta (MayaData)
+ * Richard Elling @richardelling (Viking Enterprise Solutions a division of Sanmina Corporation)
+
+*Infrastructure requests (CI / CNCF Cluster):*
+
+_Development needs:_
+
+none in the short term. We currently use Travis CI, but we may want to use CNCF resources to deploy Jenkins for node E2E test.
```

Comment:
Do you pay for Travis CI (for more concurrent job for example) or just the free version for now?

Reply:
We have setup using the free version for now.